src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -1241,327 +1305,351 @@ void GoogleCalendarSyncAdaptor::finishedRequestingRemoteEvents(int accountId, co
         return; // still waiting for more requests to finish.
```

Comment:
At line 823 of the new PR, we check to see if the m_syncSucceeded[accountId] value is true prior to attempting to use that value - so, yes, I think it's ok.